docker/ghd

```diff
@@ -19,4 +19,4 @@ cd "$(script_path)"
 
 set +e
 
-poetry run bash -c " cd \"$current_dir\" && ghd $@ "
+poetry run bash -c " cd \"$current_dir\" && ghd \"\$@\"" -- "$@"
```

Comment:
Für Bash-Noobs wie mich: was macht das?

Reply:
Das "$@" wurde vorher ausexplodiert, sodass bash mehrere Parameter bekommen hat. Das hat bei den Tests von @zyv nur funktioniert, weil er wohl nur das Kommando ohne zusätzliche Parameter beim Aufruf angegeben hat, aber bei mehreren Parametern sowas wie `bash -c '... foo' 'bar'` draus wurde. Durch diese Änderung wird das `$@` nicht hier im Skript, sondern in der Sub-Shell ausgewertet, und die Parameter hinter dem `--` dort ausexplodiert.

Reply:
Faszinierend, @stohrendorf danke dir für die korrekte Lösung

Reply:
Ist mir beim Testen auch nicht aufgefallen, da ich leider auch nur mit einem Parameter getestet habe... :(